Source/WebKit/Platform/SharedMemory.h

```diff
@@ -70,6 +71,12 @@ class SharedMemoryHandle {
         Win32Handle;
 #endif
 
+    static SharedMemoryHandle create(Type&&, size_t);
+
+    SharedMemoryHandle() = default;
```

Comment:
I think later on the default constructor, clear() and isNull() should be removed.
That might be appropriate for this commit, or then some other commit..

Reply:
I agree but am voting for separate commit as this looks to affect a number of classes and serializers.

Reply:
Ok I have `clear` removed in https://github.com/WebKit/WebKit/pull/13659 but the rest look non-trivial.

Reply:
Ok so I removed clear in https://github.com/WebKit/WebKit/commit/62bea872d1f186dcb512d2296f2e5cce1f499f94 but removing the default constructor which would be required for removing `isNull` is non-trivial and would touch a lot of the classes that needed to be modified to get all the move semantics changes.